priv/templates/phx.gen.live/components.ex

```diff
@@ -270,11 +271,14 @@ defmodule <%= @web_namespace %>.Components do
   def input(%{type: "checkbox"} = assigns) do
     ~H"""
     <label phx-feedback-for={@name} class="flex items-center gap-4 text-sm leading-6 text-zinc-600">
+      <input type="hidden" id={@id || @name} name={@name} value="false" />
```

Comment:
This will result in a "duplicate ID" error in the browser. Assuming we even need an ID on the hidden field, maybe do something like this?

```elixir
<input type="hidden" id="#{@id || @name}_hidden" name={@name} value="false" />
```

Reply:
Good one. I think it doesn't even need the id. I'll test tomorrow when I push more changes.